src/utils/api.js

```diff
@@ -0,0 +1,109 @@
+class Api {
+  constructor({baseUrl, headers}) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  getAppInfo() {
+    this.getUserInfo();
+    return Promise.all([this.getInitialCards()]);
```

Comment:
`Promise.all` is used for 2 or more promises


`this.getUserInfo()` should be inside the array

```js

  getAppInfo() {
    return Promise.all([this.getInitialCards(), this.getUserInfo()]);
  }
```

Reply:
This was just a miss on my part, this was covered in the youtube video